core/objects.go

```diff
@@ -254,6 +274,10 @@ type Challenge struct {
 
 	// Used by dns and dvsni challenges
 	Validation *jose.JsonWebSignature `json:"validation,omitempty"`
+
+	// Contains information about URLs used or redirected to and IPs resolved and
+	// used
+	ValidationRecord *ValidationRecord `json:"validationRecord,omitempty"`
```

Comment:
To be clear about the decision here: We're talking about the difference between the following two alternatives for challenge syntax:

```
1. { "type": "dvsni", "validationRecord": { "dvsni": {stuff} } }
   { "type": "simpleHttp", "validationRecord": { "simpleHttp": [{stuff}] } }
2. { "type": "dvsni", "validationRecord": [{stuff}] }
   { "type": "simpleHttp", "validationRecord": [{stuff}] }
```

My point is that the extra `dvsni` or `simpleHttp` isn't adding anything.